src/App.jsx

```diff
@@ -86,6 +88,8 @@ const App = () => {
         <Route path='/admin/book' element={<AdminRoute> <Book /> </AdminRoute>} />
```

Comment:
as suggestion use constant file for path 

export const API_BASE_URL = 'https://api.example.com';
export const USER_PROFILE_PATH = '/users/profile';
export const SETTINGS_PATH = '/settings';

Reply:
Sure I will do that, making constant file path could be a good practice.